ldclient/client.py

```diff
@@ -35,36 +39,50 @@ def __init__(self,
                  verify=True,
                  defaults=None,
                  events=True,
-                 stream_processor_class=None,
-                 feature_store_class=None,
-                 feature_requester_class=None,
+                 update_processor_class=None,
```

Comment:
This model of having an update processor + a feature requester is consistent with the other SDKs